### PR TITLE
bt-17: Add app theme

### PR DESCRIPTION
**Pull request by illia-dobro**
[Link to issue  ](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/issues/17)

### CODE REVIEW

File: mobile/src/bundles/common/styles/colors.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    primary: '#274F8D',
+    one: '#18A0FB',
+    error: '#FF2121',
+    hover: '#0084DD',
+    background: '#F7F8FC',
+    input: '#D5DCE8',
+    text2: '#90A3BF',
+    textDark: '#7C8798',
+    tag: '#DCE5FF',
+};
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/styles/colors.ts]:
```suggestion
enum Color {
    PRIMARY = '#274F8D',
    SECONDARY = '#18A0FB',
    ...
} ;
```

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/actions/runs/5926475879/job/16067962200?pr=36

File: mobile/src/bundles/common/styles/colors.ts (hunk 1 above)
Comment (katerynakostikova) [reply mobile/src/bundles/common/styles/colors.ts]: updated comment

**Review by katerynakostikova — APPROVED** [review]

**Review by anogire — APPROVED** [review]

File: mobile/src/bundles/common/styles/colors.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    PRIMARY = '#274F8D',
+    SECONDARY = '#18A0FB',
+    ERROR = '#FF2121',
+    HOVER = '#0084DD',
+    BACKGROUND = '#F7F8FC',
+    INPUT = '#D5DCE8',
+    TEXT2 = '#90A3BF',
+    TEXTDARK = '#7C8798',
+    TAG = '#DCE5FF',
+}
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/styles/colors.ts]: We discussed switching from ts enums to js enums with coaches, so let's replace with js enum 
```suggestion
const Color = {
    PRIMARY: '#274F8D',
    SECONDARY: '#18A0FB',
    ...
} as const;
```

**Review by katerynakostikova — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

File: mobile/src/bundles/common/styles/colors.ts
Comment (nikita-remeslov) [inline mobile/src/bundles/common/styles/colors.ts:1]: I think we should rename this file as we use enum for it 

`colors.enum.ts`

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Review by katerynakostikova — COMMENTED** [review]
Please move this file to bundles/common/enums/ui

**Comment by katerynakostikova** [comment]
image

Re-export it here

**Review by katerynakostikova — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]
@illia-dobro fix conflicts please
@katerynakostikova after you can merge

**Pull request merged**